apps/designer/app/designer/features/style-panel/sections/flex-child/flex-child.tsx

```diff
@@ -82,14 +93,24 @@ const FlexChildSectionAlign = (props: RenderCategoryProps) => {
 };
 
 const FlexChildSectionSizing = (props: RenderCategoryProps) => {
-  const { createBatchUpdate, currentStyle, sectionStyle } = props;
+  const {
+    createBatchUpdate,
+    currentStyle,
+    setStyle,
+    cascadedStyle,
+    inheritedStyle,
+    sectionStyle,
```

Comment:
All for has similar names but different types. 
Like setStyle is really { property: StyleValue }
cascadedStyle is container `{ property: { breakpointId, value: StyleValue } }`
inheritedStyle is container `{ property: { instanceId, value: StyleValue } }`
sectionStyle is container `{property: RenderPropertyProps;}`

So nothing here except `setStyle` is Style ;-)